lib/project/result-row.js

```diff
@@ -0,0 +1,150 @@
+const {Range} = require('atom');
+
+class LeadingContextRow {
+  constructor(rowGroup, line, separator, matchLineNumber, dy) {
+    this.group = rowGroup
+    this.data = {
+      separator,
+      line,
+      lineNumber: matchLineNumber - dy,
+      matchLineNumber,
+    }
+    this.dy = dy
```

Comment:
The name `dy` made me think this variable represented a height value in pixels, but it's actually a number of *rows*, relative to the match row, right? What do you think about renaming this to something like `rowOffset`?

Reply:
Sure, I'll change it.